src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -186,8 +165,14 @@ async function modifyPermalinks(repoPath: string) {
       );
       let permalinkLineTrimmed = permalinkLine.trim();
 
-      // check if permalink has quotation marks, if so, remove them
-      permalinkLineTrimmed = permalinkLineTrimmed.replace(/"/g, "");
+      // check if permalink has quotation marks at the start and at the end, and if so, remove them
+      assert(permalinkLineTrimmed.startsWith(`permalink:`));
+      let permalinkValue = permalinkLineTrimmed
+        .replace(`permalink:`, "")
+        .trim();
+      permalinkValue = permalinkValue.replace(/^"/g, "").replace(/"$/, "");
+      permalinkValue = permalinkValue.replace(/^'/g, "").replace(/'$/, "");
```

Comment:
Likewise, I think here we can omit `g` after the regex as well.

Reply:
Similar response, using `/^'+/` would serve the purpose.

Reply:
aa73e608bd87a515cbda8f66f5c74bb5d8555c7d

Reply:
Similar comment, why don't we use `/^'+/`?

Reply:
fair 8eb89a8f5df436138d1f9a5f676e68743f298858